app/views/users/_user.html.erb

```diff
@@ -0,0 +1,8 @@
+<% cache user do %>
```

Comment:
We don't need to cache here yet -- initial version should be pretty dependent-less 😄